webapp/views.py

```diff
@@ -113,7 +113,28 @@ def _get_resources(self):
         content = self.request.GET.get('content')
         page = int(self.request.GET.get('page', 1))
         offset = (page - 1) * self.PER_PAGE
+        search = self.request.GET.get('q')
         feed_items = {}
+        if search:
+            api_url = (
+                '{api_url}/posts?_embed'
+                '&search={search}&{resource_filter}'
+            )
+            api_url = api_url.format(
+                api_url=self.API_URL,
+                search=search,
+                resource_filter=self.RESOURCE_FILTER,
+            )
```

Comment:
I added the comment before as a reply but:
```
api_url = (...).format(...)
```
to avoid instanciate `api_url` 2 times

Reply:
Done, good spot.